_maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_ash_walker1_skyrat.dmm

```diff
@@ -1615,7 +1615,9 @@
 "WN" = (
 /obj/structure/curtain/bounty,
 /obj/structure/table,
-/turf/open/floor/iron,
+/turf/open/floor/iron{
```

Comment:
I feel like this should probably not be just a varedit, it should likely be its own tile subtype, so that it can be re-used in the future, if need be.

Reply:
I had a look at other ruin maps and they all seem to use var edited gas mixes. Not a very good practice so I've added a lavaland atmos subtype of iron tiles to the modular directory.